### PR TITLE
Run model tests with clusterless feature enabled

### LINKED ISSUE #33058: Clusterless feature is not tested in Model tests

### PR DESCRIPTION
**Pull request by mhajas**
Closes #33058
Signed-off-by: Michal Hajas <mhajas@redhat.com>

### CODE REVIEW

**Review by pruivo — COMMENTED** [review]
forgot to click submit :cry:

File: testsuite/model/pom.xml [hunk 1]
```diff
@@ -258,7 +258,7 @@
         </profile>
 
         <profile>
-            <id>jpa+remote-infinispan</id>
+            <id>jpa+clusterless</id>
```
[the remark this answers is not in the document]
Comment (mhajas) [reply testsuite/model/pom.xml:261]: I prefer sticking with one name everywhere. It isn't very clear if we use more names for one thing. What do you mean by it should be mentioned in the GHA?

File: testsuite/model/pom.xml [hunk 2]
```diff
@@ -269,8 +269,8 @@
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
                             <systemPropertyVariables>
-                                <keycloak.profile.feature.remote_cache>enabled</keycloak.profile.feature.remote_cache>
-                                <keycloak.profile.feature.multi_site>enabled</keycloak.profile.feature.multi_site>
+                                <keycloak.profile.feature.clusterless>enabled</keycloak.profile.feature.clusterless>
+                                <keycloak.profile.feature.persistent_user_sessions>disabled</keycloak.profile.feature.persistent_user_sessions>
```
[the remark this answers is not in the document]
Comment (mhajas) [reply testsuite/model/pom.xml:273]: I don't think so. Clusterless should be at the moment a feature that can be enabled regardless of the multisite feature. It has basically nothing to do with multisite. If I am not mistaken multisite feature only enables the `/lb-check` endpoint and forces usage of persistent sessions (I hope it does not force it when clusterless is enabled but I think not).

We can add clusterless+multisite as another profile, but I am not sure it is worth it.

Did I miss something?

[the remark this answers is not in the document]
Comment (pruivo) [reply testsuite/model/pom.xml:273]: I miss nothing. I refresh my memory and you're right!

**Review by pruivo — APPROVED** [review]

**Review by ahus1 — APPROVED** [review]

**Pull request merged**